spyderlib/plugins/runconfig.py

```diff
@@ -484,11 +498,32 @@ def setup_page(self):
         thisdir_layout.addWidget(thisdir_radio)
         thisdir_layout.addWidget(thisdir_bd)
 
-        wdir_layout = QVBoxLayout()
-        wdir_layout.addWidget(wdir_label)
-        wdir_layout.addWidget(dirname_radio)
-        wdir_layout.addLayout(thisdir_layout)
-        wdir_group.setLayout(wdir_layout)
+        post_mortem = self.create_checkbox(_("Enter post mortem debugging for uncaught exceptions"), 'post_mortem', False)

```

Comment:
Also, I'd like to reword the message a bit (given that our audience are mostly scientists). What about:

    Enter debugging when errors appear during execution